src/typescript-worker/bare-module-transformer.ts

```diff
@@ -266,14 +269,17 @@ export class BareModuleTransformer {
       location.version =
         (await getPackageJson())?.dependencies?.[location.pkg] ?? 'latest';
     }
-    if (location.path === '') {
-      const packageJson = await this._cdn.fetchPackageJson(location);
-      location.path = packageJson.module ?? packageJson.main ?? 'index.js';
-    }
-    if (
-      !fileExtension(location.path) ||
-      !isExactSemverVersion(location.version)
-    ) {
+    location.version = await this._cdn.resolveVersion(location);
```

Comment:
This line turns our provided version into a concrete version by interrogating a cdn?